assets/src/utils/convert_service_utils.py

```diff
@@ -3,31 +3,55 @@
 from src import logger
 from src.config import settings
 
-logger_ = logger.get_logger(__name__)
+LOGGER = logger.get_logger(__name__)
 
 
-def post_to_convert(
+class UploadError(Exception):
+    """
+    Raises when file were not uploaded
+    """
+
+
+def post_txt_to_convert(
     bucket: str, input_text, output_pdf, output_tokens
-) -> bool:
+) -> None:
     """
-    Puts file into convert service
+    Puts txt file into convert service
     """
     try:
         response = requests.post(
-            url=settings.service_convert_uri,
+            url=settings.service_convert_txt,
             json={
                 "input_text": {"bucket": bucket, "path": input_text},
                 "output_pdf": {"bucket": bucket, "path": output_pdf},
                 "output_tokens": {"bucket": bucket, "path": output_tokens},
             },
         )
         if response.status_code != 201:
-            logger_.info(
-                f"File {input_text} failed to convert: " f"{response.text}"
+            raise UploadError(
+                "File %s failed to convert: %s", input_text, response.text
+            )
+    except requests.exceptions.ConnectionError as e:
+        LOGGER.error(f"Connection error - detail: {e}")
+    LOGGER.info(f"File {input_text} successfully converted")
```

Comment:
If you get ConnectionError, than it will be logged, but there will be no errors raised.
Probably, you should raise some exception here

Reply:
I did that after previous review